bin.js

```diff
@@ -11,12 +11,21 @@ const execspawn = require('execspawn')
 const envString = require('env-string')
 const xargv = require('cross-argv')
 const crypto = require('crypto')
+const jwt = require('jsonwebtoken')
 const Insight = require('insight')
 const updateNotifier = require('update-notifier')
 const pkg = require('./package.json')
 const tarAndUpload = require('./lib/tar-and-upload.js')
 const helpFormatter = require('./lib/help-formatter.js')
 const clean = require('./lib/clean')
+const authenticate = require('./lib/authenticate.js')
+const authMethods = {
+  authenticate,
+  // for testing success, email in token test@test.com
+  simpleSuccess: () => 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyLCJlbWFpbCI6InRlc3RAdGVzdC5jb20ifQ.04BSkxtfqwws2v893h2CDJHFtc7bqn0CGmdDIGI80TA',
```

Comment:
It's a test jwt token with email

Reply:
why is it needed? If we need for testing, can we just pass it along in tests?

Reply:
there is no easy way to pass it in with test as of right now, they are needed to test failure and success cases to mock out the original authenticate which has it's own set of tests and does various stuff including opening of url in browser. Maybe if @goto-bus-stop takes over he will have a  better idea....

Reply:
(he → they pls :))

An alternative is to pass in an env var like `CLINIC_ASK_MOCK_JWT` and return that in `lib/authenticate.js`. then we can keep the jwt value itself in tests, it's slightly less invasive. still needs a few lines in authenticate.js to handle the be able to return the mocks tho. i don't think there's a way around that because it's in a subprocess.